### PR TITLE
fix:finalized exemptions for GKE & ACM

### PR DESCRIPTION
**Pull request by g-awmalik**
- Added allowed resources and exempted namespaces for the bundle to work with GKE and ACM features
- Removed the restrict-default-namespace constraint since GKE uses it as a catch-all for admin resources which is allowed by the CIS benchmark but we don't have a good way of excluding those resources right now.
- YAML formatting updates

### CODE REVIEW

File: bundles/cis-k8s-1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ metadata:
   name: restrict-clusteradmin-rolebindings
   annotations:
     # This constraint is not certified by CIS.
-    description: "Restricts use of the cluster-admin role."
-    bundles.validator.forsetisecurity.org/cis-k8s-v1.5.1: 5.1.1
+    description: Restricts use of the cluster-admin role.
+    bundles.validator.forsetisecurity.org/cis-k8s-1.5.1: 5.1.1
```
Comment (morgante) [inline bundles/cis-k8s-1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml:22]: Why removing the v? It's helpful to separate the version from the name.

File: bundles/cis-k8s-1.5.1/README.md [hunk 2]
```diff
@@ -1,12 +1,21 @@
-CIS Kubernetes Benchmark v.1.5.1
-==================================================
+# cis-k8s-1.5.1
 
 ## Description
+sample description
```
Comment (morgante) [inline bundles/cis-k8s-1.5.1/README.md:4]: We shouldn't be eliminating the description.

File: bundles/cis-k8s-1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml (hunk 1 above)
Comment (g-awmalik) [reply bundles/cis-k8s-1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml:22]: I agree but the folder doesn't have the "v" in there which complicates the copying process from the internal library. We can either take the v out from the annotation or put a "v" in the directory. I thought the annotation modification may be less obtrusive

File: bundles/cis-k8s-1.5.1/README.md (hunk 2 above)
Comment (g-awmalik) [reply bundles/cis-k8s-1.5.1/README.md:4]: Acked. That was mistake, I'll put it back in there.

File: bundles/cis-k8s-1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml (hunk 1 above)
Comment (g-awmalik) [reply bundles/cis-k8s-1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml]: Added

**Review by morgante — APPROVED** [review]

**Pull request merged**